### DISCUSSION TITLE
Configurable exception hook supressions

### DISCUSSION THREAD

**Discussion by RXminuS** [fastapi/typer#695] [Show and tell]
I'm using Typer together with Apache beam and whenever an exception occurs there's a large amount of irrelevant internal state to Beam. This leads to unmanagebly large variable outputs when using the rich logging feature.

For a similar reason this is why internally  [on this line](https://github.com/tiangolo/typer/blob/92846bb994024b7383169414e0bc250cb5311025/typer/main.py#L69) `typer` and `click_path` are supressed.

My suggestion would be to a configuration to `DeveloperExceptionConfig` so that as a user I can supply my own list of additional supressions. But before making a PR I'd just like some quick feedback if this is the right approach and location? Or maybe I missed a different way to configure this?

**Comment by peterjc** [comment]
I had the same idea on #1172 (e.g. hiding SQLalchemy details)
